mobile/lib/features/trade/trade_bottom_sheet_tab.dart

```diff
@@ -83,7 +83,7 @@ class _TradeBottomSheetTabState extends State<TradeBottomSheetTab> {
     WalletInfo walletInfo = context.watch<WalletChangeNotifier>().walletInfo;
 
     Direction direction = widget.direction;
-    String label = direction == Direction.long ? "Buy" : "Sell";
+    String label = direction == Direction.long ? "Buy / Long" : "Sell / Short";
```

Comment:
I'm against this change. Can we leave just "Buy" and "Sell" please?

Reply:
copying over from the other comment:

> sure, I can remove that change from this changeset! 
It was one of the feedback from the conference - people are *really* confused by only 'buy' and 'sell' (and asked questions like "can I *buy* Bitcoin here? what it is this?".
I took the new naming after Kollider (which has it like that), and thought that it is only an improvement - if less people get confused, but still understandable for pros. (especially that we're targetting *retail* traders).